### ISSUE TITLE
Failed to uprade to 20.04: exit code 100

### ISSUE THREAD

**Issue by jonneymendoza** [pop-os/pop#935]
**Distribution (run `cat /etc/os-release`):**
 
 
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**

**Issue/Bug Description:**
Hi,

i tried to upgrade my thinkpad x1e laptop from 19.10 to 20.04 just now and i keep getting an error from the settings/details/os-upgrade menu with this error:

``` Failed to upgrade OS: failed status@ failed to update package list for the current release@ command failed with exit status: exit code: 100```

**Steps to reproduce (if you know):**

Go to settings/details/os-upgrade and click dowload  button

**Expected behavior:**

Upgrade to 20.04

**Other Notes:**

**Comment by isamu1024** [comment]
Got the same error, i had to remove a broken ppa and it upgrades flawlessly.

**Comment by arielsierra** [comment]
I experience the same inconvenience that I can't upgrade the operating system. I get this error: 
![Screenshot from 2020-05-01 11-42-55](https://user-images.githubusercontent.com/15117925/80827838-94c2a080-8ba1-11ea-8dfe-75db38e9f6f8.png)

**Comment by mdr0id** [comment]
Seeing this too and command doesn't work either:
```
$ sudo pop-upgrade release upgrade systemd -f
Event: removing deprecated and/or conflicting packages
Event: updating package lists for the current release
Event: fetching updated packages for the current release
Event: upgrading packages for the current release
Event: ensuring that system-critical packages are installed
Event: updating the source lists to the new release
Event: updating package lists for the current release
Release upgrade status: release upgrade aborted: failed to update package lists for the new release: command failed with exit status: exit code: 100
pop-upgrade: dbus client error: calling ReleaseUpgradeFinalize method failed
```

**Comment by leviport** [comment]
@mdr0id Don't use the `-f` flag. Now that 20.04 is released, `-f` will try to grab 20.10 which doesn't exist yet.

**Comment by mdr0id** [comment]
@leviport Is the underlying script from the GUI running this same command? I ask because this is the exact error I get following: https://support.system76.com/articles/upgrade-pop/

Basically the Upgrade OS GUI is throwing this error when attempting to upgrade from a 19.10 node

**Comment by leviport** [comment]
Yes it is the same. You may still be seeing the same bug, I just wanted to eliminate the possibility of passing the wrong flag to `pop-upgrade`

**Comment by gianmarcocalbi** [comment]
> I experience the same inconvenience that I can't upgrade the operating system. I get this error:
> ![Screenshot from 2020-05-01 11-42-55](https://user-images.githubusercontent.com/15117925/80827838-94c2a080-8ba1-11ea-8dfe-75db38e9f6f8.png)

Same issue for me

**Comment by MarkoB-hub** [comment]
i also have this issue, anyone found a fix?

**Comment by damusix** [comment]
Oryx pro user. Getting same error.

**Comment by damusix** [comment]
Resolved after running `sudo dpkg --configure -a` two times

**Comment by mdr0id** [comment]
Just used the GUI from 19.04 to 20.04 and installed without prior mentioned error. TY!

**Comment by jxoesneon** [comment]
This seems to be related to where you are pulling your packages from, I was running into the same issue and after disabling all PPAs except the ones for System76, I was able to complete the process.

**Comment by Melonadev** [comment]
> Resolved after running `sudo dpkg --configure -a` two times

Yup, this works for me for installing a .deb file
